lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -80,6 +82,11 @@ def self.available_options
                                        description: 'Skips confirmation',
                                        is_string: false, # true: verifies the input is a string, false: every kind of value
                                        default_value: false), # the default value if the user didn't provide one
+          FastlaneCore::ConfigItem.new(key: :app,
+                                       env_name: 'APP',
+                                       description: 'The app to get the release version for',
+                                       is_string: true, # true: verifies the input is a string, false: every kind of value
+                                       default_value: 'wordpress'), # the default value if the user didn't provide one
```

Comment:
I don't think this is the right choice here to provide a default value of `wordpress`. This will only work on the WordPress-Android repo, and won't make sense in all other repos (WCAndroid, SNAndroid). Default values should only be used imho if they are the common value used in **most** of our client repos.

Reply:
I was debating on this, I agree let's not use wordpress as the default here 👍